# initial altitude setpoint
alt_sp: 1.2

# PID gains
alt_Kp: 1.0
alt_Ki: 0.1
alt_Kd: 0.01

# maximum vertical velocity, in absolute value, [m/s]
MAX_VUP: 2.0
MAX_VDOWN: 0.5
